### PR TITLE
feat: add voice api

### PR DESCRIPTION
**Pull request by jackshen310**
## Release Notes

- **New Features**
  - Enhanced documentation for the Coze API SDK, including a "Quick Start" section and a new "Key Features" overview.
  - Introduced the `Voices` class for managing audio voices, with methods to clone and list voices.
  - Added speech synthesis functionality with the new `Speech` class.
  - Expanded chat capabilities to handle audio events.
  - New example for audio processing and voice management in the Node.js environment.
  - Improved settings modal with voice selection and data handling enhancements.

- **Bug Fixes**
  - Improved error handling in the API error classes.

- **Tests**
  - Added unit tests for the `Voices` and `Speech` functionalities to ensure correct behavior.

- **Chores**
  - Updated configuration files and directory structures for better organization, including changes to the README for clarity.

### CODE REVIEW

File: packages/coze-js/src/resources/audio/index.ts [hunk 1]
```diff
@@ -1,8 +1,47 @@
...
+import { type RequestOptions } from '../../core.js';
 
 export class Audio extends APIResource {
   rooms: Rooms = new Rooms(this._client);
+  voices: Voices = new Voices(this._client);
+
+  /**
+   * @description Speech synthesis | 语音合成
+   * @param params
+   * @param params.input - Required. Text to generate audio | 要为其生成音频的文本
```
Comment (Tecvan-fe) [inline packages/coze-js/src/resources/audio/index.ts:14]: English first.

File: packages/coze-js/src/resources/audio/voices/voices.ts [hunk 2]
```diff
@@ -0,0 +1,119 @@
+import { type GenericFormData, toFormData } from 'axios';
+
+import { APIResource } from '../../resource.js';
```
Comment (Tecvan-fe) [inline packages/coze-js/src/resources/audio/voices/voices.ts:3]: ts & js 混用？

**Review by Tecvan-fe — APPROVED** [review]
LGTM

File: packages/coze-js/examples/node/src/voice.mjs [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      '今天天气真是太好了，阳光灿烂，心情超级棒，但是朋友最近的感情问题也让我心痛不已，好像世界末日一样，真的好为他难过。',
+    voice_id: '742894224871836***',
+  });
+  console.log('client.audio.voices.clone', voiceObj);
+} catch (error) {
+  console.error('Failed to clone voice:', error.message);
+}
+
+try {
+  const speechBuffer = await client.audio.speech({
```
Comment (chyroc) [inline packages/coze-js/examples/node/src/voice.mjs:28]: 这里改成 .speech.create
（友商是这样的）

File: packages/coze-js/src/resources/audio/index.ts (hunk 1 above)
Comment (jackshen310) [reply packages/coze-js/src/resources/audio/index.ts:14]: Same as python, see: https://github.com/coze-dev/coze-py/blob/main/cozepy/chat/__init__.py

File: packages/coze-js/src/resources/audio/voices/voices.ts (hunk 2 above)
Comment (jackshen310) [reply packages/coze-js/src/resources/audio/voices/voices.ts:3]: 这个等我们解决那个构建问题再看处理，这样写实际上是兼容 node 和 web，不过这个sdk不用支持node，所以可以不需要怎么写。

File: packages/coze-js/examples/node/src/voice.mjs (hunk 3 above)
Comment (jackshen310) [reply packages/coze-js/examples/node/src/voice.mjs:28]: OK

**Review by jsongo — APPROVED** [review]

**Pull request merged**